frontend/src/features/schedule/ScheduleCard.tsx

```diff
@@ -0,0 +1,62 @@
+import { CarIcon, DotIcon, LocationIcon, PersonIcon } from "@/assets/icons";
+import type { DrivingDataType } from "@/types/drivingType";
+import type { ScheduleType } from "@/features/schedule/Schedule.types";
+import {
+  Content,
+  ContentWrapper,
+  HeaderWrapper,
+  RouteText,
+  RouteWrapper,
+  ScheduleCardContainer,
+  UserWrapper,
+  CardTypeText,
+  TimeText,
+  TimeWrapper,
+} from "@/features/schedule/ScheduleCard.style";
+
+import { theme } from "@/styles/themes.style";
+const { color } = theme;
+const grayColor = color.GrayScale.gray4;
+const blueColor = color.SemanticScale.blue[400];
+
+interface DrivingCardProps {
+  drivingType: ScheduleType;
+  data: DrivingDataType;
+}
+const ScheduleCard = ({ drivingType, data }: DrivingCardProps) => {
+  return (
+    <div css={ScheduleCardContainer}>
+      <div css={HeaderWrapper}>
+        <div css={RouteWrapper}>
+          <p css={RouteText}>경로 #{data.routeId}</p>
+          <div css={UserWrapper(drivingType)}>
+            <PersonIcon
+              fill={drivingType === "inProgress" ? blueColor : grayColor}
+            />
+            <p>{data.totalUserCount}</p>
+          </div>
+        </div>
+        <div css={TimeWrapper}>
+          <p css={TimeText}>{data.routeStartTime}</p>
+          <p css={TimeText}>~</p>
+          <p css={TimeText}>{data.routeEndTime}</p>
```

Comment:
```suggestion
          <p css{TimeText}>{`${data.routStartTime}~${data.routEndTime}`}</p>
```
이 방법은 어떨까요 ?? gap 때문에 사용하신거라면 무시하셔도 됩니다

Reply:
해당 사항 반영해도 gap에 문제가 없는 것을 확인했습니다. 감사합니다! 👍🏻